src/python/bot/fuzzers/libFuzzer/launcher.py

```diff
@@ -706,6 +709,42 @@ def use_mutator_plugin(target_name, extra_env, chroot):
   return True
 
 
+def get_merge_directory():
+  """Returns the path of the directory we can use for merging."""
+  temp_dir = fuzzer_utils.get_temp_dir()
+  return os.path.join(temp_dir, MERGE_DIRECTORY_NAME)
+
+
+def create_merge_directory():
+  """Create the merge directory and return its path."""
+  merge_directory_path = get_merge_directory()
+  shell.create_directory(
+      merge_directory_path, create_intermediates=True, recreate=True)
+  return merge_directory_path
+
+
+def is_sha1_hash(possible_hash):
+  """Returns True if |possible_hash| looks like a valid sha1 hash."""
+  if len(possible_hash) != 40:
+    return False
+
+  return all(char in string.hexdigits for char in possible_hash)
```

Comment:
nit: might be faster to just do 

SHA1_REGEX = re.compile("^[a-fA-F0-9]{40}$") in global
and then SHA1_REGEX.match(possible_hash).

Reply:
Would rather not use regex if unnecessary. I may use a globally defined set instead though.